docs/contributing.rst

```diff
@@ -158,15 +159,17 @@ Running ``./test/isoltest`` again will result in a test failure:
             Success
 
 
-``isoltest`` prints the expected result next to the obtained result, but also provides a way to change edit / update / skip the current contract or to even quit.
+``isoltest`` prints the expected result next to the obtained result, and also
+provides a way to change edit, update, skip, or quite the current contract.
```

Comment:
typo: quite

Reply:
and I think quit just stops everything while the other three operations apply to the current contract.